pkgs/development/tools/nil/default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, rustPlatform, fetchFromGitHub }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "nil";
+  version = "0.0.0-unstable-2022-09-06";
+
+  src = fetchFromGitHub {
+    owner = "oxalica";
+    repo = pname;
+    rev = "b469b939f1ab2550cddbff1a289043c680969d84";
```

Comment:
I just created a date tag for this.

https://github.com/oxalica/nil/releases/tag/2022-09-10